lib/matplotlib/ticker.py

```diff
@@ -829,13 +829,13 @@ class LogFormatter(Formatter):
     """
     Format values for log axis.
     """
-    def __init__(self, base=10.0, labelOnlyBase=True):
+    def __init__(self, base=10.0, label_only_base=True):
```

Comment:
The easiest way to do this is `…, label_only_base=True, labelOnlyBase=None` and check whether labelOnlyBase was provided to deprecate it.